source/Makefile

```diff
@@ -132,12 +132,9 @@ OPTIM=-Os -flto -finline-small-functions -findirect-inlining -fdiagnostics-color
 # global defines ---------------------------------------------------------------
 GLOBAL_DEFINES += $(DEV_GLOBAL_DEFS) -D USE_RTOS_SYSTICK -D MODEL_$(model) -D VECT_TAB_OFFSET=$(bootldr_size)U
 
-# Without debug code
-DEBUG=
+DEBUG=-g3

```

Comment:
-g3 is equivalent to -ggdb3 , it's just that I personally prefer the latter as it's making obvious it's something GDB-specific.

Reply:
Good point.
@Ralim should I open another PR or would you just change it directly?